src/txmempool.cpp

```diff
@@ -661,26 +661,25 @@ void CTxMemPool::removeConflicts(const CTransaction &tx)
     }
 }
 
-/**
- * Called when a block is connected. Removes from mempool.
- */
 void CTxMemPool::removeForBlock(const std::vector<CTransactionRef>& vtx, unsigned int nBlockHeight)
 {
+    // Remove confirmed txs and conflicts when a new block is connected, updating the fee logic
```

Comment:
Very low relevance, but "updating the fee logic" is a nit confusing to me:
- why "updating" and not "update"? "remove ..., updating ..." suggests it's a side effect in the same step as opposed to a separate step
- is it `(updating the fee) logic` or `updating the (fee logic)`?  What logic is updated?

My suggestion: `also update fee stats` or just `also update fees`.
Otherwise LGTM.

Reply:
"Remove... while also update" -> "Remove..., updating".

If you agree with the change please comment an uppercase ACK with the latest commit hash to your message.